File: joseffe-barroso/Python/aula04/while/exercise41.py
quantity = int(input("Qual é o valor da sequencia? "))
while (quantity <= 0 or quantity > 100): 
  quantity = int(input("O valor da sequência deve ser positivo e menor que cem. Informe novamente: "))

index = 2
sequence = 2
sumNumbers = 0

while (index < quantity + 2):
  print(sequence)
  sumNumbers = sumNumbers + sequence
  sequence = index**2+1
  index = index + 1

print(f"A soma dos valores deu: {sumNumbers}")

File: joseffe-barroso/Java/src/aula04/While/Exercise41.java
package While;

import java.util.Scanner;
import java.lang.Math;

public class Exercise41 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		int n, i, soma, termo;
	
		System.out.println("Digite o termo que quer encontrar (valor deve ser positivo e menor que 100): ");
		n = scanner.nextInt();
		
		while ((n <= 0) || (n > 100)) {
			System.out.println("O valor de N deve ser positivo e menor que 100.");
			System.out.println("Digite o termo que quer encontrar (valor deve ser positivo e menor que 100): ");
			n = scanner.nextInt();
		}
		
		i = 1;
		soma = 0;
		
		while (i <=n) {
			termo = ( (int) Math.pow(i,2) + 1);
			System.out.println(i + ". " + termo);
			i++;
			soma = soma + termo;	
		} System.out.println("A soma é: " + soma);
		
		scanner.close();

	}

}